src/main/docker/container-jwt/entrypoint.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+# Copyright 2018-2025 Cyface GmbH
+# 
+# This file is part of the Cyface Data Collector.
+#
+#  The Cyface Data Collector is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  
+#  The Cyface Data Collector is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with the Cyface Data Collector.  If not, see <http://www.gnu.org/licenses/>.
+
+# Entrypoint script for the Docker container which starts the Collector API.
+#
+# author: Klemens Muthmann
+# author: Armin Schnabel
+# Version 1.0.0
+
+DEFAULT_API_PORT="8080"
+JAR_FILE="collector-all.jar"
+LOG_FILE="/app/logs/collector-out.log"
+SERVICE_NAME="Cyface Collector API"
+
+main() {
+  loadAuthParameters
+  loadApiParameters
+  loadCollectorParameters
+  loadConfig
+  waitForDependency "mongo" 27017
+  startApi
+}
+
+loadApiParameters() {
+  if [ -z "$CYFACE_API_PORT" ]; then
+    CYFACE_API_PORT=$DEFAULT_API_PORT
+  fi
+
+  if [ -z "$CYFACE_API_HOST" ]; then
+    CYFACE_API_HOST="localhost"
+  fi
+
+  if [ -z "$CYFACE_API_ENDPOINT" ]; then
+    CYFACE_API_ENDPOINT="/api/v4/"
+  fi
+}
+
+loadAuthParameters() {
+  if [ -z "$CYFACE_AUTH_TYPE" ]; then
+    CYFACE_AUTH_TYPE="oauth"
+  fi
+  if [ -z "$CYFACE_JWK" ]; then
+    echo "Unable to find JWK. Please set the environment variable CYFACE_JWK to an appropriate value! API will not start!"
+  fi
+
+  echo "Using Auth type: $CYFACE_AUTH_TYPE"
+  echo "Using JWK $CYFACE_JWK"
```

Comment:
I wouldn't log secrets, but the JWT shouldn't be a secret afaik so this should be ok